packages/rs-drive-abci/src/execution/platform_events/core_based_updates/update_quorum_info/v0/mod.rs

```diff
@@ -50,39 +52,36 @@ where
             return Ok(()); // no need to do anything
         }
 
+        // We should request the quorum list from 8 blocks behind the core block height
```

Comment:
But we passing current core block height isn't it?

Reply:
We need input from core, I believe originally we were requesting 8 blocks before, but then changed things

Reply:
Could you please update or remove it?